locales/en.json

```diff
@@ -1284,8 +1284,8 @@
       "product_duplicate_sku_alert": "There are multiple products with SKU \"{0}\". Please choose one to add to cart:",
       "product_was_not_added_alert": "Products with following SKUs were not added to the cart: {0}",
       "data_is_invalid_alert": "The details you provided are invalid.",
-      "manually_tab": "Enter Manually",
-      "copy_n_paste_tab": "Copy and Paste"
```

Comment:
@NaMax66 Do you mean keep "Copy and Paste"? "&" is on design and it will not fit into mobile tabs otherwise (with "Upload file"). Even more : I was needed a slightly narrow spaces between controls and reduce "Upload file" too, because designer accidentally used spaces for layout larger, than in our theme